Updated the clean URLs lesson to use dynamic routes (#365)
zeit/next-site: www/pages/learn/basics/clean-urls-with-dynamic-routing/finally.mdx

import Layout from '../../../../components/learn/Layout';

export const meta = {
  title: 'Clean URLs with Route Masking',
  courseId: 'basics',
  lessonId: 'clean-urls-with-dynamic-routing',
  stepId: 'finally'
};

## Finally

> This is all for this lesson, if you want to know more about dynamic routing check out the [dynamic-routing example](https://github.com/zeit/next.js/tree/canary/examples/dynamic-routing) in the Next.js repo and the [documentation](https://nextjs.org/docs#dynamic-routing).

Reviewer: ```suggestion
That's the end of this lesson, if you want to know more about dynamic routing check out the [dynamic-routing example](https://github.com/zeit/next.js/tree/canary/examples/dynamic-routing) in the Next.js repo and the [documentation](https://nextjs.org/docs#dynamic-routing).
```

> Our next lesson is all about dynamic data fetching. Let's add some data now.

Reviewer: ```suggestion
Our next lesson is about dynamic data fetching. Let's add some data now.
```
